tools/changelog-helper.sh

```diff
@@ -126,10 +126,15 @@ check_or_add_pr() {
         return
     fi
     checked_ids[$id]=1
-    local json=$(gh pr view "${id/#/}" --json title,author)
+    if grep -qE "#$id\>" <<<"$changelog"; then
```

Comment:
Can you add a comment on what this does and why, please.

Reply:
And, perhaps - like the "not merged" below - echo a message.

Reply:
> Can you add a comment on what this does and why, please.

Done.

> And, perhaps - like the "not merged" below - echo a message.

Hm, I'm not convinced. It was generally nice to see that repeated invocations of this script added less and less stuff (e.g. when run again later in the release process). If we generate a line of output of each PR even if it was already there, the output becomes harder to read, IMO. If you feel strongly, I can add it though.

Reply:
Yeah, that's why I was hesitant.  Probably best without.